wordpress-mobile/WordPress-iOS
# Comments: show No Connection view

Ref: #15955 

This uses the `NoResultsViewController` to display the `Unable to load this content right now` view when there is no connection. It is displayed for filters that do not have cached results. Comments are still displayed for the last fetched filter.

To test:

---
- Go to Site > Comments.
- Disable connection.
- Select a different Comment filter.
- Verify the no connection view is displayed.
- Verify the Comments are still displayed on the original filter.

![no_connection](https://user-images.githubusercontent.com/1816888/111380254-b4ec3880-8669-11eb-8385-c2a5ce1cbd6d.jpeg)


---
- Enable connection.
- Tap `Retry`.
- Verify Comments for the selected filter are displayed.

---
- Select a filter with no Comments.
- Disable connection.
- Select a different Comment filter.
- Verify the no connection view is displayed.
- Verify the no results view is still displayed on the original filter.

| ![no_results](https://user-images.githubusercontent.com/1816888/111380369-df3df600-8669-11eb-8de0-bd72a8186a4f.jpeg) | ![no_connection](https://user-images.githubusercontent.com/1816888/111380393-e82ec780-8669-11eb-806e-c876ac5a3b85.jpeg) |
|--------|-------|

---
PR submission checklist:
